src/tree/registerTrees.ts

```diff
@@ -76,5 +77,8 @@ export function registerTrees(): void {
         await ext.volumesTree.refresh(node);
     });
 
-    registerCommand('vscode-docker.openUrl', async (_context: IActionContext, node: OpenUrlTreeItem) => node.openUrl());
+    ctx.subscriptions.push(
```

Comment:
I don't think `ctx` is needed here. The rest of the tree registrations simply use `ext.context.subscriptions.push`, we should do the same here.